### DISCUSSION TITLE
Migration to new cluster which has difference between SnapshotClass driver!!!

### DISCUSSION THREAD

**Discussion by phucnv282** [vmware-tanzu/velero#6428] [Community support Q&A] [answered]
Hello all,
I'm trying Velero to migrate K8s from on-prem to GKE.
In on-prem I'm using Ceph CSI and successfully backup PVC to GCS. Next step, I wanna restore those data into the GKE cluster, but it uses Google CSI instead of Ceph. (We don't have SnapshotClass with ceph.csi driver anymore)
Is this plan possible? I know Velero allows changing Storage Class (it's excellent), but how about SnapshotClass?
If I can do that, can you give me some guides? If I am wrong, fix me with an alternative possible solution. Many thanks!!!

**Comment by draghuram** [comment, marked as answer]
To restore from one type of storage to another, you need to do file system backup and use storage class mapping at restore time. It is not possible to take Ceph snapshot from source cluster and create a GKE type PVC. It is not just about Volume snapshot class.

**Reply by phucnv282** [reply]
Yeah, thank you for your point out. I also just realize that truth: https://velero.io/docs/main/file-system-backup/
Thank u so much!
